packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -1241,6 +1257,6 @@ export function client(
   withGcEnabled?: boolean
 ): MultiClientSpecBuilder {
   const specBuilder = new MultiClientSpecBuilder();
-  specBuilder.withGCEnabled(withGcEnabled === true);
+  specBuilder.ensureManualLruGC();
```

Comment:
Shouldn't this be:

```ts
if (! withGcEnabled) {
  specBuilder.ensureManualLruGC();
}
```

Reply:
Oh yeah, thanks for checking this. 

I intended to delete withGcEnabled all together, because multitab will always have lru gc. I just forgot to delete it.

Reply:
see: https://github.com/firebase/firebase-js-sdk/pull/7234